devicon.json

```
@@ -9520,6 +9520,29 @@
         "color": "#F05138",
         "aliases": []
     },
+    {
+        "name": "swiper",
+        "altnames": [],
+        "tags": [
+            "framework",
+            "open-source"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "plain"
```

This field should be the same as the version in the filename, so here it would be original, since the filename is swiper-original.svg
```suggestion
                "original"
```

---

Okay. Changing it.